client/src/app/admin/panel/tickets/admin-panel-search-tickets.js

```diff
@@ -28,8 +30,13 @@ function updateSearchTicketsFromURL() {
     const currentPathName = window.location.pathname;
     const currentSearch = window.location.search;
     const currentPath = `${currentPathName}${currentSearch}`;
+
     if(currentPath.includes(SEARCH_TICKETS_PATH)) {
         searchTicketsUtils.getFiltersFromParams().then((listConfig) => {
+            const showFilters = (currentSearch !== SEARCH_TICKETS_QUERY) && queryString.parse(currentSearch).custom;
```

Comment:
@LautaroCesso when you change query filters (except page/orderby), `custom` parameter gets deleted; right?